src/platform/packages/shared/kbn-esql-ast/src/visitor/global_visitor_context.ts

```diff
@@ -382,7 +391,18 @@ export class GlobalVisitorContext<
     return this.visitWithSpecificContext('visitChangePointCommand', context, input);
   }
 
-  // Expression visiting -------------------------------------------------------
+  public visitForkCommand(
+    parent: contexts.VisitorContext | null,
+    node: ESQLAstChangePointCommand,
+    input: types.VisitorInput<Methods, 'visitForkCommand'>
+  ): types.VisitorOutput<Methods, 'visitForkCommand'> {
+    const context = new contexts.ChangePointCommandVisitorContext(this, node, parent);
+    return this.visitWithSpecificContext('visitChangePointCommand', context, input);
```

Comment:
Same

Reply:
Good catch, done in https://github.com/elastic/kibana/pull/216743/commits/cbec7502379cd4de94f4f88ffea9ae783e012772